### PR TITLE
Beginning DBM recommendations docs.

### PR DESCRIPTION
**Pull request by amw-zero**
### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

**Comment by drichards-87** [comment]
Hi @amw-zero. It looks like you're still working on this PR. I'm going to add the Work in progress label to remove it from the Docs on call queue. You can remove the label once it's ready to be reviewed. Thanks!

File: content/en/database_monitoring/_index.md
Comment (drichards-87) [inline content/en/database_monitoring/_index.md:111]:
```suggestion
The [Recommendations page][6] highlights problems and optimization opportunities, helping you save time by prioritizing what's most important.
```

File: content/en/database_monitoring/recommendations/index.md
Comment (drichards-87) [inline content/en/database_monitoring/recommendations/index.md:6]:
```suggestion
---
```

File: content/en/database_monitoring/recommendations/index.md
Comment (drichards-87) [inline content/en/database_monitoring/recommendations/index.md:14]:
```suggestion
Choose your database type to view the supported recommendations:
```

File: content/en/database_monitoring/recommendations/index.md
Comment (drichards-87) [inline content/en/database_monitoring/recommendations/index.md:71]:
```suggestion
Choose your hosting type to view the supported recommendations:
```

File: content/en/database_monitoring/recommendations/index.md
Comment (drichards-87) [inline content/en/database_monitoring/recommendations/index.md:6]: To confirm, this feature is in Preview, right? If yes, we can add a banner that states it's in Preview.

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left some feedback and a couple of questions.

File: content/en/database_monitoring/recommendations/index.md
Comment (amw-zero) [reply content/en/database_monitoring/recommendations/index.md:4]: We're sending out a GA announcement, so I'll actually remove the Preview banner from the UI instead.

File: content/en/database_monitoring/_index.md
Comment (amw-zero) [reply content/en/database_monitoring/_index.md]: Done.

File: content/en/database_monitoring/recommendations/index.md
Comment (amw-zero) [reply content/en/database_monitoring/recommendations/index.md]: I applied this to all of the other recommendation descriptions as well.

File: content/en/database_monitoring/recommendations/index.md
Comment (amw-zero) [reply content/en/database_monitoring/recommendations/index.md]: I added some more description.

This feature is enabled by default, and there's no way to disable it.

File: content/en/database_monitoring/recommendations/index.md
Comment (amw-zero) [reply content/en/database_monitoring/recommendations/index.md]: I moved the Low Disk Space recommendation back into the DBMS tabs, and just added a note that it is only supported on RDS for now. I think the hosting type selection is too confusing at the moment since it only affects this one recommendation.

File: config/_default/menus/main.en.yaml
Comment (drichards-87) [reply config/_default/menus/main.en.yaml:3923]: FYI, this is something we'll still want to address. In order to have the Recommendations article appear above Troubleshooting, the weight will need to be changed to 13, and the weights for the other articles that appear on the same level in the left nav will also need to be incremented.

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left some additional feedback on the PR.

File: config/_default/menus/main.en.yaml
Comment (amw-zero) [reply config/_default/menus/main.en.yaml:3923]: I see, I thought this was font weight.

**Comment by amw-zero** [comment]
Great suggestions. I made all of the suggested changes.

**Review by drichards-87 — APPROVED** [review]
@amw-zero PR looks good. Do we need to wait for additional reviews before merging it?

**Comment by amw-zero** [comment]
Yea I just need an approval from someone on DBM. @joelmarcotte ?

**Comment by drichards-87** [comment]
> Yea I just need an approval from someone on DBM. @joelmarcotte ?

Sounds good. We have merge queue enabled on the repo now, so you should be able to merge it after someone on the DBM team approves the PR.

**Pull request merged**
